sys/sys/signal.h

```diff
@@ -310,8 +310,8 @@ typedef	struct __siginfo {
 } siginfo_t;
 
 #ifdef _KERNEL
-#ifdef COMPAT_CHERIABI
-struct siginfo_c {
+#ifdef COMPAT_FREEBSD64
+struct siginfo_64 {
```

Comment:
We already have siginfo64 with the correct name below, and this current one is wrong, just dropping __capability is insufficient, pointers need to be uint64_t.